docs/index.rst

```diff
@@ -62,3 +59,22 @@ General Documentation
    getting_started
    tutorials/index
    api
+
+
+.. _astroplan_authors:
+
+Authors
+=======
+
+Maintainers
```

Comment:
I just found the title "maintainer" odd. To me "maintainer" means (among other things) fixing bugs, reviewing pull requests, doing releases, i.e. the grunt work that's needed to get a package started and going, not so much implementing new features.
So at least until now I'd say "lead developer" characterises the role by Jazmin and Brett much better. Although now that GSoC is over, it could be that they become the maintainers in the sense that I understand the role, and I see that http://www.astropy.org/affiliated/ also uses "maintainer".

So @eteq – whatever you decide to put here is fine with me!